packages/js/product-editor/src/components/block-editor/block-editor.tsx

```diff
@@ -284,7 +277,7 @@ export function BlockEditor( {
 			 * If the product form template is not available, use the block instances.
 			 * ToDo: Remove this fallback once the product form template is stable/available.
 			 */
-			const editorTemplate = productFormTemplate ?? blockInstances;
+			const editorTemplate = blockInstances ?? productFormTemplate;
```

Comment:
`blockInstances` are the first option when rendering the template. Otherwise it would show the editor with the simple from which is not ready to use yet.

Reply:
updated :-)